adafruit_atecc/adafruit_atecc_asn1.py

```diff
@@ -38,7 +38,7 @@
 import struct
 
 # pylint: disable=invalid-name
-def get_signature(signature, data):
+def get_signature(signature: str, data: str) -> None:
```

Comment:
Looking at how `signature` expects anything accessed via index to be an integer, it looks like it needs to implement the [buffer protocol](https://docs.circuitpython.org/en/latest/docs/reference/glossary.html#term-buffer-protocol) but with `int`.  CircuitPython has a the `ReadableBuffer` type from the `circuitpython_typing` library available to use for this.  You can then import it and use it here for `signature`.  Let me know if you need help with that!

Reply:
`data` is the same, since it looks like we're addition `bytes` objects to it.